Sekoia.io/sekoiaio/triggers/cases.py

```diff
@@ -150,6 +161,12 @@ def handle_event(self, message):
         case_attrs = message.get("attributes", {})
         event_type: str = message.get("type", "")
         event_action: str = message.get("action", "")
+        filter_notifications = [
+            self._filter_by_mode(message),
+            self._filter_by_priority(message),
+            self._filter_by_assignees(message),
+            self._filter_by_uuids(message),
+        ]
```

Comment:
Filters must be applied on the case, not on the notification.
Can you move this after you retrieve the case from the API and update the `filter_by_*` methods to work with the [response](https://docs.sekoia.io/xdr/develop/rest_api/alert/#tag/Cases/operation/get_case_resource) of the API?